nginx-1.21.4-wolfssl.patch

```diff
@@ -123,7 +239,7 @@ index 84afecd0..fe7e328e 100644
  
      /*
       * OpenSSL 1.0.2+ allows configuring a curve list instead of a single
-@@ -1599,10 +1617,26 @@ static int
+@@ -1599,10 +1656,26 @@
```

Comment:
Am I ready this patch right... removes `static int`?

Reply:
No...that's just `diff` 's attempt to tell you what function the change is in.